Kernel/Arch/aarch64/Processor.cpp

```diff
@@ -109,8 +109,12 @@ void Processor::initialize(u32)
         asm volatile("wfi");
 }
 
-void Processor::flush_tlb_local(VirtualAddress, size_t)
+void Processor::flush_tlb_local(VirtualAddress vaddr, size_t page_count)
 {
+    // We must flush the data cache here, such that the updated page tables actually
+    // end up in main memory, and thus actually get used by the MMU.
+    Aarch64::Asm::flush_data_cache(vaddr.get(), PAGE_SIZE * page_count);
+
```

Comment:
Do note that this commit is *incorrect* and causes crashes on bare metal when unmapping pages. So this cannot be merged as-is!